frontend/pages/schedule/ManageSchedulePage/ManageSchedulePage.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useState, useCallback, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import { push } from "react-router-redux";
+import { IQuery } from "interfaces/query";
+import { IGlobalScheduledQuery } from "interfaces/global_scheduled_query";
+// @ts-ignore
+import globalScheduledQueryActions from "redux/nodes/entities/global_scheduled_queries/actions";
+// @ts-ignore
+import queryActions from "redux/nodes/entities/queries/actions";
+// @ts-ignore
+import { renderFlash } from "redux/nodes/notifications/actions";
+
+import paths from "router/paths";
+import Button from "components/buttons/Button";
+import ScheduleError from "./components/ScheduleError";
+import ScheduleListWrapper from "./components/ScheduleListWrapper";
+import ScheduleEditorModal from "./components/ScheduleEditorModal";
+import RemoveScheduledQueryModal from "./components/RemoveScheduledQueryModal";
+
+const baseClass = "manage-schedule-page";
+
+const renderTable = (
+  onRemoveScheduledQueryClick: React.MouseEventHandler<HTMLButtonElement>,
+  allGlobalScheduledQueriesList: IGlobalScheduledQuery[],
+  allGlobalScheduledQueriesError: any,
+  toggleScheduleEditorModal: any
+): JSX.Element => {
+  if (Object.keys(allGlobalScheduledQueriesError).length !== 0) {
+    return <ScheduleError />;
+  }
+
+  return (
+    <ScheduleListWrapper
+      onRemoveScheduledQueryClick={onRemoveScheduledQueryClick}
+      allGlobalScheduledQueriesList={allGlobalScheduledQueriesList}
+      toggleScheduleEditorModal={toggleScheduleEditorModal}
+    />
+  );
+};
+interface IRootState {
+  entities: {
+    global_scheduled_queries: {
+      isLoading: boolean;
+      data: IGlobalScheduledQuery[];
+      errors: any;
+    };
+    queries: {
+      isLoading: boolean;
+      data: IQuery[];
+    };
+  };
+}
+
+const ManageSchedulePage = (): JSX.Element => {
+  const dispatch = useDispatch();
+  const { MANAGE_PACKS } = paths;
+  const handleAdvanced = (event: any) => dispatch(push(MANAGE_PACKS));
```

Comment:
Delete argument since it is unused
```suggestion
  const handleAdvanced = () => dispatch(push(MANAGE_PACKS));
```